sdk/src/claim.rs

```diff
@@ -288,6 +306,37 @@ impl Claim {
         }
     }
 
+    /// Create a new claim with a user supplied GUID.
+    /// user_guid: is user supplied guid conforming the C2PA spec for manifest names
+    /// claim_generator: User agent see c2pa spec for format
+    pub fn new_with_user_guid(claim_generator: &str, user_guid: &str) -> Self {
```

Comment:
These parameters could be <S: Into<String>> and user_guid.into(). so we don't force cloning.